website_recaptcha_v2/models/website.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-FileCopyrightText: 2010-2014 Elico Corp
+# SPDX-FileContributor: Augustin Cisterne-Kaas <augustin.cisterne-kaas@elico-corp.com>
+# SPDX-FileCopyrightText: 2015 Tech-Receptives Solutions Pvt. Ltd.
+# SPDX-FileCopyrightText: 2019 Simone Orsi - Camptocamp SA
+# SPDX-FileCopyrightText: 2019 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import requests
+
+from odoo import _, api, fields, models
+
+URL = "https://www.recaptcha.net/recaptcha/api/siteverify"
```

Comment:
Would this be a better naming ?

```suggestion
RECAPTCHA_API_URL = "https://www.recaptcha.net/recaptcha/api/siteverify"
```